guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -33,8 +33,9 @@ Use a FQDN, not an IP address:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-registration \
---foreman-proxy-templates \
+# {installer-scenario-smartproxy} \
```

Comment:
@ekohl Is it necessary to use the scenario in this case?

Reply:
Verified with @evgeni that "scenario" is required only with the initial installation. Removing it.